tests/Auth0Test.php

```diff
@@ -84,6 +84,34 @@ public function testThatExchangeReturnsFalseIfNoCodePresent()
         $this->assertFalse( $auth0->exchange() );
     }
 
+    /**
+     * Test that the exchanges fails when there is not a stored nonce value.
+     *
+     * @throws ApiException
+     * @throws CoreException
+     */
+    public function testThatExchangeFailsWithNoStoredNonce()
```

Comment:
I assume this is the case because we don't allow to opt-out sending a `nonce` during authentication?

Reply:
Correct. Code exchange would only happen during login.